src/deepsparse/yolo/pipelines.py

```diff
@@ -245,8 +268,15 @@ def process_engine_outputs(
 
         batch_boxes, batch_scores, batch_labels = [], [], []
 
-        for image_output in batch_output:
-            batch_boxes.append(image_output[:, 0:4].tolist())
+        original_image_shapes = kwargs.get("original_image_shapes")
+        for idx, image_output in enumerate(batch_output):
+            boxes = image_output[:, 0:4]
+            if original_image_shapes:
+                boxes = self._scale_boxes(
+                    boxes=boxes,
+                    original_image_shape=original_image_shapes[idx],
+                )
+            batch_boxes.append(boxes.tolist())
```

Comment:
I'd rather not clutter the "main" logic here and just change one single line to:
```suggestion
            batch_boxes.append(self.scale_boxes(image_output[:, 0:4], original_image_shapes = ....).tolist())
```

Reply:
Updated